pages/index.tsx

```diff
@@ -187,12 +186,12 @@ export default function Home({ isConnected }) {
   )
 }
 
-export async function getServerSideProps(context) {
-  const client = await clientPromise
-
-  const isConnected = await client.isConnected()
-
-  return {
-    props: { isConnected },
-  }
-}
+// export async function getServerSideProps(context) {
+//   const client = await clientPromise
+//
+//   const isConnected = await client.isConnected()
+//
+//   return {
+//     props: { isConnected },
+//   }
+// }
```

Comment:
Do we need this file? I thought this service only _serves_ through a backend API with no frontend.

Reply:
It's NextJS boilerplate, so no we don't technically need it